### ISSUE TITLE
Error "error: Error executing process: 'command': No such file or directory"

### ISSUE THREAD

**Issue by brunoluigi** [CrunchyData/bridge-cli#151]
I'm trying to use cb cli in a Devcontainer but I keep getting the following error:

```
vscode ➜ /workspaces/linkana-deploy (main) $ cb login
error: Error executing process: 'command': No such file or directory
   id: f67e04975ae64b4287d8ea0e905f12bc
vscode ➜ /workspaces/linkana-deploy (main) $ 
```

Devcontainer OS: Debian GNU/Linux 11 (bullseye)

Reinstallation log:
```
vscode ➜ /workspaces/linkana-deploy (main) $ brew reinstall CrunchyData/brew/cb
==> Fetching crunchydata/brew/cb
==> Downloading https://github.com/CrunchyData/bridge-cli/archive/refs/tags/v3.4.3.tar.gz
Already downloaded: /home/vscode/.cache/Homebrew/downloads/b171eeeea7067496670a845da03c271d1ac223801ab6f912dd7f00163def1b47--bridge-cli-3.4.3.tar.gz
==> Reinstalling crunchydata/brew/cb 
==> make build RELEASE=1
🍺  /home/linuxbrew/.linuxbrew/Cellar/cb/3.4.3: 7 files, 6.5MB, built in 1 minute 58 seconds
==> Running `brew cleanup cb`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
```

**Comment by abrightwell** [comment]
@brunoluigi - I've been able to reproduce this one in a DevContainer. I'll work on getting a fix into our next update which shouldn't be too long from now.

Also, FWIW, if you want to pull the pre-built binaries instead of using Homebrew, then those are available in the [releases](https://github.com/CrunchyData/bridge-cli/releases) and can be installed in your DevContainer using the `gh-release` feature:

```
{
    "image": "debian:bullseye",
    "features": {
        "ghcr.io/devcontainers-contrib/features/gh-release:1": {
            "repo": "CrunchyData/bridge-cli",
            "binaryNames": "cb"
        }
    }
}
```

Thanks!

**Comment by brunoluigi** [comment]
Wow, that was quick!!! Thanks @abrightwell!!!

Quick feedback. Using the ENV var worked for me, I followed this guide to set everything up nicely: https://code.visualstudio.com/remote/advancedcontainers/environment-variables#_option-2-use-an-env-file

**Comment by brunoluigi** [comment]
> @brunoluigi - I've been able to reproduce this one in a DevContainer. I'll work on getting a fix into our next update which shouldn't be too long from now.
> 
> Also, FWIW, if you want to pull the pre-built binaries instead of using Homebrew, then those are available in the [releases](https://github.com/CrunchyData/bridge-cli/releases) and can be installed in your DevContainer using the `gh-release` feature:
> 
> ```
> {
>     "image": "debian:bullseye",
>     "features": {
>         "ghcr.io/devcontainers-contrib/features/gh-release:1": {
>             "repo": "CrunchyData/bridge-cli",
>             "binaryNames": "cb"
>         }
>     }
> }
> ```
> 
> Thanks!

That's awesome! I was using Homebrew just to install the cb cli and it was taking a while!

**Comment by abrightwell** [comment]
Fixed in `v3.4.4`.
